framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -9576,6 +9750,16 @@ VulkanAddressReplacer& VulkanReplayConsumerBase::GetDeviceAddressReplacer(const
     return it->second;
 }
 
+bool VulkanReplayConsumerBase::UseExtraDescriptorInfo(const VulkanDeviceInfo* device_info) const
+{
+    return options_.dumping_resources || UseAddressReplacement(device_info);
+}
+
+bool VulkanReplayConsumerBase::UseAddressReplacement(const VulkanDeviceInfo* device_info) const
+{
+    return !device_info->allocator->SupportsOpaqueDeviceAddresses();
```

Comment:
Ok, in case the previous comment got deleted, my concern is that should we check this per situation?  For example, the ray tracing opaque address support might not be present, but the other standard one is, but we never use ray tracing in the current app so it doesn't matter.